hail/src/main/scala/is/hail/expr/ir/EmitStream.scala

```diff
@@ -1249,7 +1254,7 @@ object EmitStream {
           mux(cond,
             emitStream(thn, env).map(TypedTriplet(t, _)),
             emitStream(els, env).map(TypedTriplet(t, _))
-          ).map(_.untyped)
+          ).map(_.untyped(t))
```

Comment:
why did this change?

Reply:
Ah, that was a issue left over from the rebase.  It doesn't have to.  I fixed it.